src/main/java/task/TaskTodo.java

```diff
@@ -0,0 +1,13 @@
+package task;
+
+import time.Time;
+
+public class TaskTodo extends TaskBase {
+
+    public TaskTodo(TaskType taskType, Time taskTime, String taskDescription) {
+        super(taskType , taskTime, taskDescription);
+    }
+
+
+
```

Comment:
delete empty lines

Reply:
Thank you for your comments, will look into those!